pkg/testutils/resourcebuilder/consts.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourcebuilder
+
+const (
+	// OpenshiftMachineAPINamespaceName is the name of the OpenShift
+	// Machine API namespace.
+	OpenshiftMachineAPINamespaceName = "openshift-machine-api"
+
+	testClusterIDValue = "cpms-cluster-test-id"
```

Comment:
We probably don't want this to be `cpms` anymore, where else is it used?

Reply:
As machine label, machine selector.  I have dropped the cpms prefix.